sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveExternalCatalog.scala

```diff
@@ -189,11 +189,11 @@ private[spark] class HiveExternalCatalog(conf: SparkConf, hadoopConf: Configurat
       throw new TableAlreadyExistsException(db = db, table = table)
     }
     // Before saving data source table metadata into Hive metastore, we should:
-    //  1. Put table provider, schema, partition column names, bucket specification and partition
-    //     provider in table properties.
+    //  1. Put table provider, location URI, schema, partition column names, bucket specification
+    //     and partition provider in table properties.
```

Comment:
Could we create a separate function for [this step](https://github.com/cloud-fan/spark/blob/97f04cf246e70e411a2693232e5a54f7542a17f2/sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveExternalCatalog.scala#L206-L248)? The related codes become longer and longer.